### PR TITLE
Lagom 1.6.1

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: lightbend-telemetry/lightbend-telemetry-java-mvn/pom.xml [hunk 1]
```diff
@@ -45,7 +45,7 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.6.0</version>
+                    <version>1.6.1</version>
```
Comment (ihostage) [inline lightbend-telemetry/lightbend-telemetry-java-mvn/pom.xml]: Miss :smile: It is not a Lagom version.

File: lightbend-telemetry/log-correlation-java-mvn/pom.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@
                 <plugin>
                     <groupId>org.codehaus.mojo</groupId>
                     <artifactId>exec-maven-plugin</artifactId>
-                    <version>1.6.0</version>
+                    <version>1.6.1</version>
```
Comment (ihostage) [inline lightbend-telemetry/log-correlation-java-mvn/pom.xml]: And here 😉

**Comment by dwijnand** [comment]
Thanks, @ihostage 😆

File: grpc-example/grpc-example-java/project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.8
```
Comment (ignasi35) [inline grpc-example/grpc-example-java/project/build.properties]: gRPC samples [used to](https://github.com/lagom/lagom-samples/pull/104) have some issue with sbt `1.3.x`. I hope it's fixed now, but we may need to bump `akka-grpc`/`play-grpc` too (or stay on `1.2.8`).

**Pull request merged**